### PR TITLE
v1.2.2

### PR DESCRIPTION
**Pull request by y0sher**

### CODE REVIEW

File: eth/contract/generate.go [hunk 1]
```diff
@@ -1,5 +1,6 @@
 // Package contract generates the contract bindings for v1 contract.
 package contract
 
+// contract.abi is taken from https://github.com/bloxapp/ssv-network/blob/contract-abi/docs/dev/v1.0.0.rc5/abi/SSVNetwork.json
```
Comment (y0sher) [inline eth/contract/generate.go:4]: is this the current version of the contract to use for next release? @mtabasco , @moshe-blox , @lior-blox , before deploying this we should make sure prod and is aligned with this.

Comment (y0sher) [reply eth/contract/generate.go:4]: @moshe-blox @mtabasco @lior-blox 
*NOTE*: this version doesn't seem to be the version on mainnet / prod. we should probably update here to be the correct version.

Comment (y0sher) [reply eth/contract/generate.go:4]: latest contract version is this - https://github.com/bloxapp/ssv-network/blob/contract-abi/docs/testnet/v1.1.0/abi/SSVNetwork.json
we should probably update.

Comment (y0sher) [reply eth/contract/generate.go:4]: https://github.com/bloxapp/ssv/pull/1254

**Review by moshe-blox — APPROVED** [review]

**Review by olegshmuelov — APPROVED** [review]

**Pull request merged**